neptunecontrib/monitoring/sacred.py

```diff
@@ -100,9 +100,19 @@ def started_event(self, ex_info, command, host_info, start_time, config, meta_in
 
     def completed_event(self, stop_time, result):
         if result:
-            neptune.log_metric('result', result)
+            if isinstance(result, tuple): # logging multiple results
+                for i, r in enumerate(result):
+                    if isinstance(r, float):
+                        neptune.log_metric(f'result_{i}', r)
```

Comment:
I think this is the reason why it failed to build on python 3.5 @david1309

Could you please change it to:

```python
neptune.log_metric('result_{}'.format(i), r)
```

Also `.format()` is used in other places of this lib so it would be consistent.